tests/e2e/specs/editor/media/insertMediaFromLibrary.js

```diff
@@ -33,5 +38,7 @@ describe('Inserting Media from Media Library', () => {
     await expect(page).toMatchElement(
       '[data-testid="frameElement"]:nth-of-type(2)'
     );
+
+    await percySnapshot(page, 'E2E: Inserting Media from Media Library');
```

Comment:
I don't think you need to use "E2E: " prefix everywhere, not that you're on a different project.

Reply:
Indeed 👍